src/main/rx/spektrum.c

```diff
@@ -43,6 +43,7 @@
 #define SPEKTRUM_1024_CHANNEL_COUNT 7
 
 #define SPEK_FRAME_SIZE 16
+#define SPEKTRUM_TIME_NEEDED_PER_FRAME 5000
```

Comment:
This is actually interframe/iterbyte delay. Line must be at idle between packets.